app/src/main/java/com/example/pricingpal/model/modules/SupabaseModule.kt

```diff
@@ -65,12 +82,12 @@ object SupabaseModule {
      * client for Supabase obtained from the SupabaseClient.
      *
      */
-    @Provides
-    @Singleton
-    fun provideSupabaseGoTrue(client: SupabaseClient): GoTrue {
+    //@Provides
```

Comment:
Did you mean to keep this commented out code? Also, was it not necessary for the goTrue authorization framework?

Reply:
I went ahead and removed it. From what I can see, the new provideSupabaseAuth handles it fine. It's pulling from the same library (io.github.jan.supabase.gotrue), just in a way that aligns more cleanly with the examples in the documentation.